wd_capabilities.au3

```diff
@@ -1,5 +1,11 @@
+#include-once
+
+; standard UDF's
 #include <MsgBoxConstants.au3>
+
+; WebDriver related UDF's
 #include "wd_helper.au3" ; https://github.com/Danp2/WebDriver
```

Comment:
Don't think this is needed

Reply:
I take a look in next few hours.

Reply:
done

Reply:
This line still needs removed from wd_capabilities AFAICS